homeassistant/components/landisgyr_heat_meter/config_flow.py

```diff
@@ -94,8 +96,9 @@ async def validate_ultraheat(self, port: str):
         reader = UltraheatReader(port)
         heat_meter = HeatMeterService(reader)
         try:
-            async with async_timeout.timeout(10):
+            async with async_timeout.timeout(30):
```

Comment:
Isn't it better to move the timeout value to the `const.py`? Future changes will be easier if needed

Reply:
Agreed. Did this. Tnx!